docs/how-tos/configure-teleop-workspace.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Configure a teleop workspace"
+linkTitle: "Configure a teleop workspace"
+weight: 10
+type: "docs"
+description: "Create and configure a teleop workspace with widgets."
+images: ["/how-tos/teleop/full-workspace.png"]
+icon: true
+tags: ["teleop", "configuration"]
+languages: []
+viamresources: ["sensor", "camera", "movement sensor"]
+platformarea: ["viz", "data"]
+level: "Intermediate"
+date: "2024-11-13"
+# updated: "2024-08-26"  # When the tutorial was last entirely checked
+cost: "0"
+---
+
+You can use teleop to create a custom workspace where you can visualize data from a machine.
+Follow this guide to learn how to create a custom workspace using teleop, visualizing data from a camera, a sensor, or a movement sensor.
+
+{{% alert title="In this page" color="info" %}}
+
+- [Configure a workspace](#configure-a-workspace)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{% expand "A configured machine with teleoperable components" %}}
+
+Make sure your machine has at least one of the following:
+
+- A [camera](/components/camera/)
+- A [movement sensor](/components/movement-sensor/)
+- A [sensor](/components/sensor/)
+
+See [configure a machine](/how-tos/configure/) for more information.
+
+{{% /expand%}}
+
+## Configure a workspace
+
+{{< table >}}
+{{% tablestep %}}
+**1. Create a workspace in the Viam app**
+
+Log in to the [Viam app](https://app.viam.com/).
+
+Navigate to the **FLEET** page's **TELEOP** tab.
+Create a workspace by clicking **+ Create workspace**.
+Give it a name.
+
+{{<imgproc src="/how-tos/teleop/blank-workspace.png" resize="800x" style="width: 500px" class="fill aligncenter" declaredimensions=true alt="Blank teleop page.">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Add widgets**
+
+You are prompted to add a widget to add teleop visualizations to your workspace.
```

Comment:
Unless you mean that pulsing animation, there is nothing prompting anyone to add a widget.
```suggestion
```

Reply:
I did mean the pulsing animation but I can see how that's a small detail that would be ignored